doc/distrib/NodeHelpFiles/_review/Revit.Elements.Parameter.GroupType.md

```diff
@@ -0,0 +1,12 @@
+## In Depth
+`Parameter.GroupType` returns the given parameter's GroupType. 
+
+For non-shared parameters parameter groups are predefined in Revit, while shared parameters allow for the definition of custom groups. For more information on working with shared parameters and groups, visit this help article: [Create Shared Parameter Files, Groups, and Parameters](https://help.autodesk.com/view/RVT/2025/ENU/?guid=GUID-94EA2B8E-2C00-4D29-8D5A-C7C6664DE9CE)
```

Comment:
Love the additional context! But, this statement is a little hard to read (especially for a non native english speaker). Suggest something like below so it reads better: "In Revit, parameter groups for non-shared parameters are predefined, whereas shared parameters allow you to define custom groups. For more information on working with shared parameters and groups, visit this help article: [Create Shared Parameter Files, Groups, and Parameters]